tools/serverpod_cli/bin/create/create.dart

```diff
@@ -19,12 +19,25 @@ const _defaultPorts = <String, int>{
   'Redis server': 8091,
 };
 
+bool isNameValid(String name) {
```

Comment:
In my opinion this one better on string_validators.dart because We also have the similar checking over there.